WcaOnRails/db/migrate/20180104132335_update_preferred_formats_for_feet.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class UpdatePreferredFormatsForFeet < ActiveRecord::Migration[5.1]
+  def change
+    PreferredFormat.where(event_id: "333ft").delete_all
+    unless Rails.env.test?
```

Comment:
This is surprising to me, why do we not want to run this on the testing db?

Reply:
Argh, this was just because my db was messed up after playing with some migrations and I ran into some issue.
We don't actually need to have an exception!